tests/vmi_servers.go

```diff
@@ -32,6 +32,13 @@ func StartHTTPServer(vmi *v1.VirtualMachineInstance, port int) {
 	HTTPServer.Start(vmi, port)
 }
 
+func StartFedoraHttpServer(vmi *v1.VirtualMachineInstance, port int) {
+	console.LoginToFedora(vmi)
```

Comment:
In most cases, the login occurs in advance and embedding it in a specific command is a re-login. If in the code there was no login before calling this command, I think an explicit one can be added to allow this.

Reply:
All the other server in this file are explicit to cirros and have a login method.
I just followed the existing pattern.
If you wish to change the pattern, it should be done in another PR,

Reply:
This is new code. Why do you prefer to use a wrong pattern?
It also has to do with the caller, not with the others do it.

Reply:
done.